.github/workflows/pr_build.yaml

```diff
@@ -28,7 +28,8 @@ jobs:
       DOTNET_GENERATE_ASPNET_CERTIFICATE: false
       DOTNET_ADD_GLOBAL_TOOLS_TO_PATH: false
       DOTNET_MULTILEVEL_LOOKUP: 0
-
+      VSIX_VERSION: 1.0.${{ github.run_number }}
```

Comment:
Does this mean the VSIX version will always be 1.0.x until I changed it in this file to be 2.0.x for instance? The AppVeyor script I use takes the version number from the .vsixmanifest instead so there's just a single source of truth. Would that be possible here too?

Reply:
Yes in this iteration -- I must have missed the script part that used the manifest version as a starting point, I'll look at it.